src/vehicle.cpp

```diff
@@ -952,38 +954,49 @@ void vehicle::use_controls( const tripoint &pos )
         add_toggle( _( "chimes" ), keybind( "TOGGLE_CHIMES" ), "CHIMES" );
         add_toggle( _( "fridge" ), keybind( "TOGGLE_FRIDGE" ), "FRIDGE" );
         add_toggle( _( "recharger" ), keybind( "TOGGLE_RECHARGER" ), "RECHARGE" );
-        add_toggle( _( "plow" ), keybind( "TOGGLE_PLOW" ), "PLOW" );
-        add_toggle( _( "reaper" ), keybind( "TOGGLE_REAPER" ), "REAPER" );
-        add_toggle( _( "planter" ), keybind( "TOGGLE_PLANTER" ), "PLANTER" );
-        add_toggle( _( "scoop" ), keybind( "TOGGLE_SCOOP" ), "SCOOP" );
-        add_toggle( _( "water purifier" ), keybind( "TOGGLE_WATER_PURIFIER" ), "WATER_PURIFIER" );
+        if( veh->is_furniture ) {
+            add_toggle( _( "plow" ), keybind( "TOGGLE_PLOW" ), "PLOW" );
+            add_toggle( _( "reaper" ), keybind( "TOGGLE_REAPER" ), "REAPER" );
+            add_toggle( _( "planter" ), keybind( "TOGGLE_PLANTER" ), "PLANTER" );
+            add_toggle( _( "scoop" ), keybind( "TOGGLE_SCOOP" ), "SCOOP" );
+            add_toggle( _( "water purifier" ), keybind( "TOGGLE_WATER_PURIFIER" ), "WATER_PURIFIER" );
+        }
```

Comment:
You hide plows and reapers for non-furniture? Why would a tractor be a furniture?

Reply:
Oops, that if shouldn't be there at all.